module/extension/Director/src/Director.cpp

```diff
@@ -211,9 +211,9 @@ namespace module::extension {
 
         // A task has arrived, either it's a root task so we send it off immediately, or we build up our pack for when
         // the Provider has finished executing
-        on<Trigger<DirectorTask>, Sync<DirectorTask>>().then(
+        on<Trigger<BehaviourTask>, Sync<BehaviourTask>>().then(
```

Comment:
This is where the weird boundary is.
In theory, there is a Behaviour DSL, and the Director is an algorithm that operates with it.
So this is the director working with a BehaviourTask it has received from the DSL

Reply:
So you mean Provide, Stop, Start, When, etc etc are Behaviour DSL?
I don't like it being called Behaviour though since that word is already used for something else. It may cause confusion and delay.
What about Action? That is a term used in the subsumption system which I assume is all disappearing once this is finished.